packages/react/src/components/LocalNav/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as LocalNav } from "./LocalNav";
+export { default as LocalNavArgs } from "./LocalNav.args";
```

Comment:
@wc-lei This is probably why you're getting a rollup error. We should export the typs in `LocalNav.props.ts` but we should not export the test data in `LocalNav.args.ts`